subsys/net/lib/fota_download/include/fota_download_util.h

```diff
@@ -31,6 +33,82 @@ extern "C" {
 int fota_download_parse_dual_resource_locator(char *const file, bool s0_active,
 					     const char **selected_path);
 
+
+/**@brief Internal Fota download utils init.
+ *
+ * @param client_callback Callback for monitoring download state.
+ * @param smp_image_type Use true for DFU SMP target otherwise use false.
+
+ *
+ * @retval 0 If successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int fota_download_client_init(fota_download_callback_t client_callback, bool smp_image_type);
```

Comment:
Why doesn't this function have the same prefix as the others? `fota_download_util_*`

Reply:
I can change that